Final submission: Handling multiple services in a monorepo using Docker

lucianozapata opened the pull request:
# Final Submission

## Title

Handling multiple services in a monorepo using Docker

## Names and KTH ID

- Luciano Zapata (zapata@kth.se)
- Bror Sebastian Sjövald (sjovald@kth.se)

## Deadline

Task 2

## Category

Presentation

## Description

Our presentation will explain how to handle several docker containers using Docker compose in a monorepo. We will briefly discuss the benefits of using monorepos and how Docker compose can be used to easily build and handle a local environment.
 

**This PR includes the Presentation & Mentimeter slides.**

Commit:
Task 2: Presentation

BrorSebastianSjovald committed:
Merge branch 'KTH:2022' into 2022

Commit:
Final submission for presentation

javierron commented:
Hello @lucianozapata thanks for the submission! 

We are taking presentation submissions as links in the existing readme file, instead of creating new files. Thanks!

lucianozapata committed:
Merge branch 'KTH:2022' into 2022

Commit:
fix: added link to slides in README

Commit:
refactor

lucianozapata commented:
Hello @javierron !
Alright, I changed so that the Links are now in the README & I removed the pdf's in the folder.
Thank you!

Let me know if anything is missing 👍
